composer.json

```diff
@@ -45,9 +45,14 @@
         "apigen": [
             "php devtools/vendor/bin/apigen -c build/config/apigen.neon -d build/api -s src -s vendor/symfony/config -s vendor/symfony/console -s vendor/symfony/yaml -s vendor/guzzle/guzzle/src -s vendor/psr/log --report build/logs/checkstyle-apigen.xml --exclude *Test.php --exclude */Tests/* --google-analytics UA-40398434-1"
         ],
+        "install-box": [
+            "curl -LSs http://box-project.github.io/box2/installer.php | php",
+            "mv box.phar ${BOX_BIN}",
+            "chmod +x ${BOX_BIN}"
+        ],
         "box": [
             "mkdir -p build/artifacts",
-            "composer update --no-dev --no-interaction",
+            "composer update --no-dev --no-interaction --prefer-lowest --prefer-stable",
```

Comment:
It doesn't fetch the packages twice, they're locally cached. But I agree we should dedicated job, because we should run tests with the same composer flags. I added an env.